docs/data_ingestion.md

```diff
@@ -6,12 +6,13 @@ The data ingestion process involves transforming input data from various sources
 
 ### Steps
 1. The first step of this is usually bringing the raw data into an AWS environment that is accessible to our ingestion workflows.
-2. Once the data is available, create an ingestion config to include the metadata and file paths for data.
-3. The source data needs to be transformed into a standardized format that id uniform across its different datasets.
-4. The transformed data is validated against a set of rules to ensure that it is in the correct format.
-5. The validated data is then ingested into the database, to be surfaced by the GraphQL API.
-6. Sync the data to the production environment.
-7. Update the database in the production environment.
+2. Once the data is available, create an ingestion **config.yaml** to include the metadata and file paths for data.
+3. Validate the **config.yaml** file to ensure that it is in the correct format.
```

Comment:
Could you elabourate to add the ability to validate the source data using the config?

Reply:
```suggestion
3. Validate the **config.yaml** file to ensure that it is in the correct format, and the source data is accessible and valid.
```

Reply:
That looks good to me!